### PR TITLE
Delete local files + renamed models

### PR DESCRIPTION
**Pull request by ningning621**
## Overview

- Deleted all the local files in our .gitignore file, mostly `.idea` and `.gradle` files
- Renamed Trip -> Route, Route -> Direction

### CODE REVIEW

**Review by aasthashah999 — APPROVED** [review]
Looks good!

File: app/src/main/java/com/example/ithaca_transit_android_v2/states/SearchState.kt [hunk 1]
```diff
@@ -38,13 +38,13 @@ class RouteDisplayState(
...
  */
-class RouteOptionState(
+class SearchRouteOptionState(
     startLocation: Location,
     endLocation: Location
 ) : SearchState(startLocation, endLocation) {}
 
 /*
- * In the Route Options state, when the user clicks either the start or end location,
+ * In the Direction Options state, when the user clicks either the start or end location,
```
Comment (Haichen-Wang) [inline app/src/main/java/com/example/ithaca_transit_android_v2/states/SearchState.kt:47]: I think the state should be [SearchRouteOptionState]

**Review by Haichen-Wang — COMMENTED** [review]
One small detail

**Review by Omarrasheed — APPROVED** [review]
Looks good lets ship

**Review by JiangoJ — APPROVED** [review]

**Pull request merged**